### ISSUE TITLE
mark dungeon and elite quests

### ISSUE THREAD

**Issue by 720720** [AeroScripts/QuestieDev#1055]
you could improve questies usability by adding (dungeon) and (elite) to the quest titles

**Comment by BreakBB** [comment]
Yes, we will add this feature very soon. It will most likely become something like this in the questlog:

- My Quest (60)
- My Quest (60+)
- My Quest (60D)
- My Quest (60R)

**Comment by O4epegb** [comment]
Would be awesome feature, waiting for this!
